ci-operator/step-registry/gitops-operator/tests/gitops-operator-tests-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -x
+
+unset CI
+
+exit_code=0
+scripts/openshift-CI-kuttl-tests.sh
+kubectl kuttl test test/openshift/e2e/sequential --config test/openshift/e2e/sequential/kuttl-test.yaml --report xml || exit_code=1
+
+
+# Move results to Artifacts directory
+find . -type f -name "*.xml"
```

Comment:
Was this command used to debug or is it there for logging purposes?

Reply:
This was for debugging initially but I left it there in case we change the naming convention in future. I hope that's ok?